README.md

```diff
@@ -1,10 +1,55 @@
 # MuckRock
+**MuckRock** &middot; [Squarelet][squarelet] &middot; [DocumentCloud][documentcloud] &middot; [DocumentCloud-Frontend][documentcloudfrontend]
 
 [![Codeship Status for MuckRock/muckrock][codeship-img]][codeship]
 [![codecov.io][codecov-img]][codecov]
 
 MuckRock is a non-profit collaborative news site that gives you the tools to keep our government transparent and accountable.
 
+## Prerequisites 
+MuckRock depends on Squarelet for user authentication. As the services need to communivate directly, the development environment for MuckRock depends on the development environment for Squarelet - the MuckRock docker containers will join Squarelet's docker network. [Please install Squarelet and set up its development environment first][squarelet].
+
+### Squarelet Integration
+The Squarelet project provides an authentication system for MuckRock. Therefore, there must be certain things set up within Squarelet before you can begin accessing them individually. 
+
+ 1. With the Squarelet containers running, in your terminal within the Squarelet folder, open up the bash shell with `inv sh`
+ 2. Within the bash shell, utilize this command to create your RSA key `./manage.py creatersakey`
+ 3. Then create a superuser utilizing the following command: `./manage.py createsuperuser`. 
+ 4. Exit the bash shell with the command `exit`
+ 5. Enter the Django shell with the command `inv shell`
```

Comment:
Creating a super user should be set to is_staff by default.  I am not sure why this was inserted into the README, but I believe steps 5 - 8 are all unnecessary and were user error on part of the developer who edited the README.  Should confirm that it works properly without these steps.

Reply:
That actually makes more sense, originally i'd thought that as well, but was unsure. removed and edited